app/playbooks/deploy.d/container-storage/container-storage.yml

```diff
@@ -54,3 +54,51 @@
       delay: 15
       register: apply
       until: apply is success
+
+    - name: wait for storage class to be ready
+      k8s_info:
+        api_version: v1
+        kind: StorageCluster
+        name: ocs-storagecluster
+        namespace: openshift-storage
+        wait: yes
+        wait_condition:
+          type: Available
+
+    - name: ensure cephfs is the default storage class
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: StorageClass
+          metadata:
+            name: ocs-storagecluster-cephfs
```

Comment:
I would make RBD the default storage class - why did you chose cephFS?

Reply:
Ah I see... because of the registry that wants RWX storage...
It is interesting, but they don't appear to have a storage class field in the registry config configMap... BUT we could create a PVC and then use its name... or set and reset the default storage class

what do you think?

Reply:
Good point. This has been changed. :+1: